enterprise/server-enterprise/src/main/java/org/neo4j/server/enterprise/EnterpriseServerSettings.java

```diff
@@ -48,4 +51,12 @@
     @Description( "Configure the policy for outgoing Neo4j Browser connections." )
     public static final Setting<Boolean> browser_allowOutgoingBrowserConnections =
             setting( "browser.allow_outgoing_connections", BOOLEAN, TRUE );
+
+    @Description( "Publicly discoverable bolt+routing:// URI to use for Neo4j Drivers wanting to access a cluster " +
+            "that this instance is a member of." )
+    @DocumentedDefaultValue( "Defaults to empty in single-instance deployments, and a bolt+routing://-schemed URI " +
+            "of the advertised address of the first found bolt connector in clustered deployments." )
+    public static final Setting<URI> bolt_routing_discoverable_address =
+            setting( "dbms.discoverable_bolt_routing_address", Settings.URI, "" );
```

Comment:
Can this setting also be made internal + unsupported for now?
Docs should also say that it is only useful for core CC instances.